run-medley

```diff
@@ -172,9 +172,15 @@ export PATH="$MAIKODIR"/`osversion`.`machinetype`:"$oldpath"
 
 cd "$OLDPWD"
 
-echo "sysout is $LDESRCESYSOUT"
-echo "running `which $prog` $geometry $screensize $mem"
-echo "start $LDEINIT"
+whichprog=`which $prog`
+
+if [ -z "$whichprog" ] ; then
```

Comment:
`which` isn't the best choice here -- it's a c-shell-ism. One alternative is
```
if command -v $prog > /dev/null 2>&1; then
    # we're good
else
   # we're not
fi
```

Reply:
trying to fix #271

Reply:
I don't know what you mean by command? Not part of my Linux.

Reply:
It's literally the word "command"